tensorflow/buildspec-2-7.yml

```diff
@@ -41,82 +41,82 @@ context:
-  #     /Dockerfile.ec2., *DEVICE_TYPE ]
-  #   context:
-  #     <<: *TRAINING_CONTEXT
-  # BuildTensorflowExampleGpuCu112Py38TrainingDockerImage:
-  #   <<: *TRAINING_REPOSITORY
-  #   build: &TENSORFLOW_GPU_TRAINING_PY3 false
-  #   image_size_baseline: &IMAGE_SIZE_BASELINE 7738
-  #   base_image_name: BuildTensorflowEC2GpuCu112Py38TrainingDockerImage
-  #   device_type: &DEVICE_TYPE gpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   cuda_version: &CUDA_VERSION cu112
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION,
-  #     "-", *OS_VERSION, "-ec2-example" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /example,
-  #     /Dockerfile., *DEVICE_TYPE ]
-  #   context:
-  #     <<: *TRAINING_CONTEXT
-  # BuildTensorflowSageMakerCpuPy38TrainingDockerImage:
-  #   <<: *TRAINING_REPOSITORY
-  #   build: &TENSORFLOW_CPU_TRAINING_PY3 false
-  #   image_size_baseline: &IMAGE_SIZE_BASELINE 4489
-  #   base_image_name: BuildTensorflowEC2CpuPy38TrainingDockerImage
-  #   device_type: &DEVICE_TYPE cpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker"
-  #   ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.sagemaker.,
-  #                        *DEVICE_TYPE ]
-  #   context:
-  #     <<: *TRAINING_CONTEXT
-  # BuildTensorflowSageMakerGpuPy38Cu112TrainingDockerImage:
-  #   <<: *TRAINING_REPOSITORY
-  #   build: &TENSORFLOW_GPU_TRAINING_PY3 false
-  #   image_size_baseline: &IMAGE_SIZE_BASELINE 12000
-  #   base_image_name: BuildTensorflowEC2GpuCu112Py38TrainingDockerImage
-  #   device_type: &DEVICE_TYPE gpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   cuda_version: &CUDA_VERSION cu112
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION,
-  #                "-", *OS_VERSION, "-sagemaker" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION,
-  #                        /Dockerfile.sagemaker., *DEVICE_TYPE ]
-  #   context:
-  #     <<: *TRAINING_CONTEXT
+  BuildTensorflowEC2CpuPy38TrainingDockerImage:
+    <<: *TRAINING_REPOSITORY
+    build: &TENSORFLOW_CPU_TRAINING_PY3 false
+    image_size_baseline: &IMAGE_SIZE_BASELINE 4489
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py38
+    os_version: &OS_VERSION ubuntu20.04
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2"
```

Comment:
The framework version on the training images should be 2.7.1, and on inference images is 2.7.0.